src/progress/Progress.ts

```diff
@@ -0,0 +1,54 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode } from '@dojo/widget-core/interfaces';
+import { ThemedMixin, ThemedProperties, theme } from '@dojo/widget-core/mixins/Themed';
+import { v } from '@dojo/widget-core/d';
+import * as css from './styles/progress.m.css';
+
+/**
+ * @type ProgressProperties
+ *
+ * Properties that can be set on a Progress component
+ *
+ * @property value            The current value
+ * @property output...........A function used to determine the output display
+ * @property showOutput.......Toggles visibility of progess bar output
+ * @property max..............Value used to calculate percent width
+ */
+export interface ProgressProperties extends ThemedProperties {
+	value: number;
+	output?(value: number, percent: number): string;
+	showOutput?: boolean;
+	max?: number;
```

Comment:
Although I expect `min` will almost always be 0, we should probably still support `min`

Reply:
ok, not a problem